python/imzml_reader.py

```diff
@@ -87,7 +88,7 @@ def to_columnar(self, dtype="uint32"):
 
         return coords_df.join(intensities_df)
 
-    def to_array(self):
+    def asarray(self):
```

Comment:
The idea is to suggest something like numpy `asarray`? Mixed feelings, because it violates our normal conventions, but whatever you think will be least confusing in the long run.